debian/control

```diff
@@ -15,7 +15,7 @@ Description: This package contains general utilities for the Openswitch project.
 
 Package: libopx-common-dev
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends},
+Depends: ${misc:Depends},
```

Comment:
It's a substitution variable to brings in shared-library dependencies, but for this package it is not defined. Since it's not defined, the build will generate a warning message when the Debian package gets created. It should be removed.